packages/devtools/lib/src/memory/memory.dart

```diff
@@ -826,14 +843,18 @@ class MemoryScreen extends Screen with SetStateMixin {
     return instance;
   }
 
+  void updateInstancesTree() {
+    _inboundTree.update();
+  }
+
   void select(InboundsTreeNode rowNode) async {
     ga.select(ga.memory, ga.inspectInstance);
 
     // User selected a new instance from the list of class instances so the
     // instance view which would be the third child needs to be removed.
     removeInstanceView();
 
-    if (rowNode == null) return;
+    if (rowNode == null || rowNode.instance == null) return;
```

Comment:
could you do `rowNode?.instance == null` instead here?

Reply:
Done.